CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 ### Bug fixes
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
+- [SDN controller] fix private network creation: do not propose bond slave PIFs (Fixes https://github.com/xcp-ng/xcp/issues/300) (PR [4633](https://github.com/vatesfr/xen-orchestra/pull/4633))
```

Comment:
I'm not sure this is clear enough for users: can users who encountered understand that it's now fixed?

Reply:
This does not fix broken network, it prevents from creating new one.

Reply:
I ~~stole your~~ found a better phrasing :)